src/main/java/duke/task/EventTask.java

```diff
@@ -0,0 +1,34 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A task that is happening on the specified date.
+ */
+public class EventTask extends Task {
+    private LocalDate d;
```

Comment:
Consider using a more descriptive name for this attribute.
```suggestion
    private LocalDate date;
```